app/src/main/java/woowacourse/omok/NotationDao.kt

```diff
@@ -0,0 +1,88 @@
+package woowacourse.omok
+
+import android.content.Context
+import android.database.Cursor
+import android.database.sqlite.SQLiteDatabase
+import android.provider.BaseColumns
+import androidx.core.content.contentValuesOf
+import woowacourse.omok.FeedReaderContract.FeedNotation
+import woowacourse.omok.FeedReaderContract.SQL_DROP_TABLE
+import woowacourse.omok.domain.omok.model.Place
+
+class NotationDao(context: Context) {
+    private val dbHelper = FeedReaderDbHelper(context)
+
+    fun save(place: Place): Place {
+        val db = dbHelper.writableDatabase
+        val id =
+            db.insert(
+                FeedNotation.TABLE_NAME,
+                null,
+                contentValuesOf(
+                    FeedNotation.COLUMN_NAME_COLOR to place.color,
+                    FeedNotation.COLUMN_NAME_ROW_COORDINATE to place.rowCoordinate,
+                    FeedNotation.COLUMN_NAME_COL_COORDINATE to place.colCoordinate,
+                ),
+            )
+        return place.copy(id = id)
```

Comment:
```suggestion
    fun save(place: Place): Place {
        val db = dbHelper.writableDatabase
        val id =
            db.insert(
                FeedNotation.TABLE_NAME,
                null,
                contentValuesOf(
                    FeedNotation.COLUMN_NAME_COLOR to place.color,
                    FeedNotation.COLUMN_NAME_ROW_COORDINATE to place.colCoordinate,
                    FeedNotation.COLUMN_NAME_COL_COORDINATE to place.rowCoordinate,
                ),
            )
        return place.copy(id = id)
```

> 이 부분은 어렵네요.. row와 col 순서를 바꾸어서 insert했다면 assertThat(actual.rowCoordinate).isEqualTo(1) 에서 실패되는게 아닌가요? 혹시 row와 col이 같은 값이 들어갔을 때를 가정하신 걸까요?

현재 이런 식으로 입력은 제대로 받았지만, dao 에서 db.insert 를 할 때 순서가 바뀌는 경우를 말한거였습니다.
지금은 이렇게 해도 `착수가_이뤄지면_기보에_기록되어야_한다` 테스트가 통과합니다.